### PR TITLE
Added qp package

### PR DESCRIPTION
**Pull request by eacharles**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by eacharles** [comment]
The scipy >=1.7.0 requirement stems from a change in the way that scipy was handling array broadcasting in the calls to the hook functions _pdf, _cdf, etc.. in classes derived from scipy.stats._rv_continuous.

File: recipes/qp/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+{% set name = "qp" %}
+{% set version = "0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
```
Comment (jaimergp) [inline recipes/qp/meta.yaml:9]: I don't think you need this line.

Comment (eacharles) [reply recipes/qp/meta.yaml:9]: No sure if  you mean line 6 or line 9.    I tried removing line 6 and it fails, (see below).

**Comment by eacharles** [comment]
It looks like setuptools_scm has migrated to using a different mechanism.   I pinned it to a pre-migration version.

File: recipes/qp/meta.yaml (hunk 1 above)
Comment (jaimergp) [reply recipes/qp/meta.yaml:9]: Ah, you need it for the SCM stuff. Ok, no probs then!

**Pull request merged**